tests/unit/ReportActionsUtilsTest.ts

```diff
@@ -318,52 +557,45 @@ describe('ReportActionsUtils', () => {
 
     describe('getLastVisibleAction', () => {
         it('should return the last visible action for a report', () => {
-            const report = {
-                ...LHNTestUtils.getFakeReport(['email1@test.com', 'email2@test.com'], 3, true),
-                reportID: 1,
+            const report: Report = {
+                ...LHNTestUtils.getFakeReport([8401445480599174, 9401445480599174], 3, true),
+                reportID: '1',
             };
-            const action = {
-                ...LHNTestUtils.getFakeReportAction('email1@test.com', 3, true),
+            const action: ReportAction = {
+                ...LHNTestUtils.getFakeReportAction('email1@test.com', 3),
                 created: '2023-08-01 16:00:00',
                 reportActionID: 'action1',
                 actionName: 'ADDCOMMENT',
                 originalMessage: {
-                    policyName: 'Vikings Policy',
-                    reason: 'policyDeleted',
-                },
-                message: {
-                    policyName: 'Vikings Policy',
-                    reason: 'policyDeleted',
+                    html: 'Hello world',
+                    whisperedTo: [],
                 },
             };
-            const action2 = {
-                ...LHNTestUtils.getFakeReportAction('email2@test.com', 3, true),
+            const action2: ReportAction = {
+                ...LHNTestUtils.getFakeReportAction('email2@test.com', 3),
                 created: '2023-08-01 18:00:00',
                 reportActionID: 'action2',
                 actionName: 'ADDCOMMENT',
                 originalMessage: {
-                    policyName: 'Vikings Policy',
-                    reason: 'policyDeleted',
-                },
-                message: {
-                    policyName: 'Vikings Policy',
-                    reason: 'policyDeleted',
+                    html: 'Hello world',
+                    whisperedTo: [],
                 },
             };
             return (
                 waitForBatchedUpdates()
                     // When Onyx is updated with the data and the sidebar re-renders
                     .then(() =>
+                        // @ts-expect-error Preset necessary values
                         Onyx.multiSet({
                             [`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`]: report,
                             [`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`]: {[action.reportActionID]: action, [action2.reportActionID]: action2},
                         }),
                     )
                     .then(
                         () =>
-                            new Promise((resolve) => {
+                            new Promise<void>((resolve) => {
                                 const connectionID = Onyx.connect({
-                                    key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`,
+                                    key: 'reportActions_test',
```

Comment:
Why this change, to signal it's a test since the key doesn't really matter here?

Reply:
It was changed to pass the ts check since  with the template string it is not recognizable as a valid key for Onyx

Reply:
@pasyukevich I just checked out your PR here, and it's working fine with `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`, could you re-check?

Reply:
@fabioh8010 reverted 

after changes  in pr this update became redundant